src/components/Header/HeaderCreateProfile.js

```diff
@@ -35,27 +35,24 @@ export default function HeaderCreateProfile({ headline }) {
 const Header = styled.section`
   display: flex;
   grid-column: 1/3;
-  margin: 20px 0;
 `
 
 const HeadlineStyled = styled.h2`
   display: inline;
   font-weight: 300;
   color: var(--font-color-white);
```

Comment:
I would just call it something like font-color-main in case you want to change it for another theme.
```suggestion
  color: var(--font-color);
```

```suggestion
  color: var(--font-color-main);
```

```suggestion
  color: var(--main-font-color);
```